attributes/default.rb

```diff
@@ -75,8 +77,9 @@
 default['openvpn']['config']['persist-key']     = ''
 default['openvpn']['config']['persist-tun']     = ''
 default['openvpn']['config']['comp-lzo']        = ''
+default['openvpn']['config']['nobind']          = ''
 
-default['openvpn']['config']['ca']              = node['openvpn']['signing_ca_cert']
+default['openvpn']['config']['ca']              = "#{node['openvpn']['signing_ca_cert']}/#{node['openvpn']['ca_cert_filename']}"
```

Comment:
Looks like this will be set by default to `/etc/openvpn/keys/ca.crt/ca.crt`.  You can probably revert this change and leave it assigned to `node[openvpn][signing_ca_cert]`